keyboards/alhenkb/macropad5x4/keymaps/numpad/keymap.c

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     * ┌───┬───┬───┬───┐
+     * │TG1│ / │ * │ - │
+     * ├───┼───┼───┼───┤
+     * │ 7 │ 8 │ 9 │   │
+     * ├───┼───┼───┤ + │
+     * │ 4 │ 5 │ 6 │   │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │   │
+     * ├───┴───┼───┤Ent│
+     * │   0   │ . │   │
+     * └───────┴───┴───┘
+     */
+    [0] = LAYOUT_num4x5(
+        TG(1),   KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,   
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_P0,     KC_PDOT
+    ),
```

Comment:
If rearranged to fit this:
```suggestion
    [0] = LAYOUT_numpad_5x4(
        TG(1),   KC_PSLS, KC_PAST, KC_PMNS,
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,   
        KC_P1,   KC_P2,   KC_P3,   
        KC_P0,     KC_PDOT,        KC_PENT
    ),
```

Then it would support the numpad_4x5 community layout. The info.json config would need to be tweaked though.